samza-azure/src/main/java/org/apache/samza/system/eventhub/consumer/EventHubSystemConsumer.java

```diff
@@ -478,6 +478,13 @@ public void onError(Throwable throwable) {
           LOG.warn(
               String.format("Received transient exception from EH client. Renew partition receiver for ssp: %s", ssp),
               throwable);
+          try {
+            // Even though these are transient exceptions, we don't want to retry too frequently which sometimes
```

Comment:
Minor: maybe rewrite this comment as
"Add a fixed delay so that we don't keep retrying when there are long-lasting failures"